speakeasy/windows/fileman.py

```diff
@@ -188,12 +189,28 @@ class FileManager(object):
     """
     Manages file system activity during emulation
     """
-    def __init__(self, config=None):
+    def __init__(self, config=None, modules=None, cmdline=None, emu=None):
         super(FileManager, self).__init__()
         self.file_handles = {}
         self.pipe_handles = {}
         self.file_maps = {}
         self.config = config
+        # self.all_modules is the modules key from the config JSON file
+        self.all_modules = modules
+
+        # This allows us to serve the emulated module for when the full
+        # path to it is not given
+        space = cmdline.find(" ")
+
+        if space != -1:
+            binname = cmdline[:space]
+        else:
+            binname = cmdline
```

Comment:
```suggestion
        binname = cmdline.partition(" ")[0]
```

Reply:
also, this isn't going to work well if there's a space in the filename. maybe use [shlex](https://docs.python.org/3/library/shlex.html) if we care about that edge case?